test/test-cases/functional/README.md

```diff
@@ -0,0 +1,10 @@
+# Functional Test Cases
+Tests to verify essential functionality using low-rate traffic (SAI PTF).
+
+## Contents
+
+| Folder/File | Description |
+| --- | --- |
+| [`saidashvnet_experimental.py`](./functional/ptf/saidashvnet_experimental.py) | Basic positive VNET-to-VNET test cases designed for running on BMv2
```

Comment:
I think you now have saidashvnet.py and saidashvnet_sanity.py - need to update table?